src/main/java/uk/gov/ons/ctp/response/action/service/ActionPlanJobService.java

```diff
@@ -98,21 +99,18 @@ private boolean hasActionPlanBeenRunSinceLastSchedule(ActionPlan actionPlan) {
    * Create a new ActionPlanJob and execute associated actions
    *
    * @param actionPlan Action plan to create and execute job for
-   * @return ActionPlanJob that was created or null if it has not been created.
    */
-  @Transactional
-  public ActionPlanJob createAndExecuteActionPlanJob(final ActionPlan actionPlan) {
+  private void createAndExecuteActionPlanJob(final ActionPlan actionPlan) {
 
     if (!actionPlanExecutionLockManager.lock(actionPlan.getName())) {
```

Comment:
I don't think you need this lock anymore, because there's a lock in the scheduler which will protect any contention from other processes